wgpu/gui/offscreen.py

```diff
@@ -38,10 +38,11 @@ def is_closed(self):
         return self._closed
 
     def _request_draw(self):
-        call_later(0, self.draw)
+        # Deliberately a no-op, because people use .draw() instead.
+        pass
 
     def present(self, texture_view):
-        # This gets called at the end of a draw pass via GPUCanvasContextOffline
+        # This gets called at the end of a draw pass via _offscreen.GPUCanvasContext
```

Comment:
Oh, this change is actually from #403, but I guess that's fine since its only a comment.